packages/npm-dependencies-validation/src/utils/fileUtil.ts

```diff
@@ -0,0 +1,25 @@
+// importing directly from `fs/promises` is not supported on nodejs 12
+import { promises, constants } from "fs";
+const { access } = promises;
+
+export function toJsonObject<T>(jsonContent: string): T {
+  try {
+    const jsonObj: T = JSON.parse(jsonContent);
+    return jsonObj;
+  } catch (error) {
+    return emptyJsonObject<T>();
+  }
+}
+
+export async function isPathExist(absPath: string): Promise<boolean> {
```

Comment:
From English grammar perspective I believe it should be: "doesPathExist"

Reply:
Done